src/libs/routing/pathnames.ts

```diff
@@ -21,7 +21,7 @@ export const PathNames = {
   trade: '/trade',
   debug: '/debug',
   createStrategy: '/strategies/create',
-  editStrategy: '/strategies/edit',
+  editStrategy: '/strategies/edit/$strategyId',
```

Comment:
interesting, this must be new in the latest router update. We should consider to apply this new way of handling params to the explorer as well.

just a note to myself, no action required.

Reply:
how is this different from /:strategyId ?

Reply:
Nothing different, just the new syntax I guess: 
https://tanstack.com/router/v1/docs/guide/path-params

Reply:
> how is this different from /:strategyId ?

we use the pathnames in the app for navigation with `Link` or `useNavigation`. The old router was not able to replace `:strategyId` or `$strategyId` with the expected variable without passing it as a function argument.

```ts
{
  ...
  explorer: (type: ExplorerType) => `/explorer/${type}`,
  ...
}
```

the new version can handle this now

```ts
<Link to="/blog/$postId" params={{ postId: '123' }}>
      Post 123
 </Link>
```

instead of

```ts
<Link to={Pathnames.blog(postId)}>
      Post 123
 </Link>
```

Reply:
https://github.com/bancorprotocol/carbon-app/issues/969